rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -76,7 +77,7 @@ enum MetricValue {
     #[serde(rename = "c")]
     Counter(f64),
     #[serde(rename = "s", deserialize_with = "encoded_series_compat_deserializer")]
-    Set(EncodedSeries<u64>),
+    Set(EncodedSeries<u32>),
```

Comment:
The reason we're changing this is because Relay actually sends the sets as buckets of u32 and not u64. Clickhouse expects u64, but we can test this out locally to make sure it's compatible and we can write data.